doc/Language/operators.pod6

```diff
@@ -680,7 +680,7 @@ Universal interface for positional access to zero or more elements of a
     say @alphabet[1];                   # OUTPUT: «b␤»
     say @alphabet[*-1];                 # OUTPUT: «z␤»
     say @alphabet[100]:exists;          # OUTPUT: «False␤»
-    say @alphabet[15, 4, 17, 11].join;  # OUTPUT: «perl␤»
+    say @alphabet[17, 0, 10, 20].join;  # OUTPUT: �raku␤»
```

Comment:
This looks weird... The rest too, come to think of it. Someone has changed that...